### DISCUSSION TITLE
essay/2022/02/18/ruby

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#165] [Comments]
# essay/2022/02/18/ruby

토익 700점 이상 정도를 받을 수 있는 언어가 몇 개 있습니다. C, C++, C#, Ruby

https://jeho.page/essay/2022/02/18/ruby.html

**Comment by ryu9663** [comment]
멋있으십니다

**Comment by voidFunctionReturn0** [comment]
Elixir 검토해보시면 좋아하실지도 모르겠습니다.
Rails와 비슷하게, Phoenix라는 웹 프레임워크도 있습니다 ㅎㅎ

**Reply by BenjaminKim** [reply]
엘릭서와 피닉스도 물론 알고 있습니다만.. 공부하게 될 날이 올지는 잘 모르겠네요. 😀
